src/components/dropdown/dropdown.stories.svelte

```diff
@@ -0,0 +1,113 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Dropdown from './dropdown.svelte'
+  import {
+    getNonStyleArgs,
+    getStyleFromArgs
+  } from '../../../.storybook/argHelper'
+  import Icon from '../icon/icon.svelte'
+
+  const countries = {
+    'NZ': 'New Zealand',
+    'AU': 'Australia',
+    'UK': 'United Kingdom',
+    'CA': 'Canada'
+  }
+</script>
+
+<Meta
+  title="Dropdown"
+  component={Dropdown}
+  argTypes={{
+    placeholder: {
+      defaultValue: 'select...'
+    }
+  }}
+/>
+
+<Template let:args>
+  <div class="container" style={getStyleFromArgs(args)}>
+    <Dropdown value={undefined} {...getNonStyleArgs(args)}>
+      <leo-option value="one">
+        <div>One</div>
+      </leo-option>
+      <leo-option value="two">Two</leo-option>
+      <leo-option>Three</leo-option>
+    </Dropdown>
+  </div>
+</Template>
+
+<Story name="Default" />
+
+<Story name="Left Icon" let:args>
+  <div class="container" style={getStyleFromArgs(args)}>
+    <Dropdown {...getNonStyleArgs(args)}>
+      <div slot="left-icon">
+        <Icon name="check-circle-outline" />
+      </div>
+      <leo-option>One</leo-option>
```

Comment:
Ideally, I think we'd specify options with `option` elements but they don't support styling or any complex children. If I `clone` the content of the option, it won't be in the shadow DOM any more (and won't get it's styles from outside the shadow).

I could pass in an element like a `<div>` here, and it would work, but then you can't specify a `<div value="foo">` without triggering intellisense errors.

My solution has just been to use a made up element (which is treated like a div).

What do you think?

Reply:
I don't think we need to worry too much about supporting `option` in this component, since we will have a select-like component with Input (which will support `option`), and the whole point of using this Dropdown component vs the regular Select component I imagine is only for complex children and properties. What do you think?

Reply:
Sweet, I guess what I have is fine for now then :smile: